mailchimp3/entities/campaignactions.py

```diff
@@ -115,6 +115,17 @@ def send(self, campaign_id):
         return self._mc_client._post(url=self._build_path(campaign_id, 'actions/send'))
 
 
+    def resend(self, campaign_id):
+        """
+        Resend a MailChimp campaign to those that did not open previously.
```

Comment:
Documentation: Please replace this description with the one provided by the API docs. `Creates a Resend to Non-Openers version of this campaign. We will also check if this campaign meets the criteria for Resend to Non-Openers campaigns.`

Reply:
Thank you for the feedback! Per your instructions, I've updated the description to match the API docs and updated the README files and setup.py to reflect Python version 3.0.13.